integration-tests/src/tests/client/features/chunk_validation.rs

```diff
@@ -71,8 +78,25 @@ fn test_chunk_validation_basic() {
         num_block_producer_seats_per_shard: vec![8; num_shards],
         gas_limit: 10u64.pow(15),
         transaction_validity_period: 120,
+        // Needed to completely avoid validator kickouts as we want to test
+        // missing chunks functionality.
+        block_producer_kickout_threshold: 0,
+        chunk_producer_kickout_threshold: 0,
+        // Needed to distribute full non-trivial reward to each validator if at
```

Comment:
This is very nice! Very happy to see that the missing chunks scenario is being tested using real epoch transition validator rewards!